### ISSUE TITLE
Calculate and warn if lanes:forward + lanes:backward != lanes

### ISSUE THREAD

**Issue by cicku** [openstreetmap/iD#8964]
### Description

Is it a good idea to be proposed here?

**Comment by 1ec5** [comment]
The validator would be capable of warning about this situation. To be accurate, the validation rule would also need to factor in other subkeys such as `lanes:both_ways` and `lanes:reversible`.

Unfortunately, the warning message would have to tell the user to muck around with raw keys, because there’s no field for directional subkeys of `lane`: #8630 #387. Validator warnings are normally for things the UI can help the user with.

**Comment by tyrasd** [comment]
yeah, as mentioned above: as long as we don't have a field (or proper UI for this, see #3822, #387), there is currently relatively little a normal mapper can do about such a warning.

Therefore, for now this is better handled in external QA tools such as osmose, or maproulette, etc.

 Once iD has better support for lanes in general, it would make sense to reconsider this.
